app/models/presenters/rvi_cdna_xp_presenter.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Presenters
+  # Used for RVI cDNA XP plates. Used for a custom 'Manual Transfer' state transition.
+  # N.B. 'Manual Transfer' is used for the RVI BCL pipeline for the cDNA XP Plate because the plates
+  # are inaccessible in the bed at the time of transfer so cannot be transferred via
+  # bed verifications. Also we cannot set its state to 'Passed' on initial bed verification
+  # because there are steps on other plates that need to be completed before the cDNA XP Plate can be transferred.
+  # and we do not want to misreport the plate being passed if an issue arises with the steps before hand.
+  class RviCdnaXpPresenter < PlatePresenter
+    include Statemachine::Shared
+
+    # Same state machine as permissive plate presenter, but with a custom 'Manual Transfer' state transition.
+    state_machine :state, initial: :pending do
+      event :take_default_path, human_name: 'Manual Transfer' do
+        transition pending: :started
```

Comment:
should there be 2 transitions here? if this presenter is for one specific plate shouldn't there by only one state change by manual transfer? or is the first backup for if the robot (bed verification) is not available for the first state transition?

Reply:
Yeah i included started: :passed in the case the bed verification cant be completed. But that should never happen so probably safe to remove?